orbyter-ml-dev/README.md

```diff
@@ -35,30 +120,30 @@ Pin `ipython==7.20.0` to avoid `jedi` autocompletion bug with earlier versions
 #### Python package updates
 
 ##### Updated
-* boto3==1.17.10
```

Comment:
Everything below here hasn't changed, it's just that my markdown linter changed all the bullets from asterisks to dashes.